src/main/java/com/example/quily/repositories/ShortLinkRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.example.quily.repositories;
+
+import com.example.quily.model.ShortLink;
+import org.springframework.data.r2dbc.repository.R2dbcRepository;
+
+public interface ShortLinkRepository extends R2dbcRepository<ShortLink, ShortLink> {
+}
```

Comment:
repository takes first datatype of model and second data type of primary key
```suggestion
public interface ShortLinkRepository extends R2dbcRepository<ShortLink, Long> {
}
```